web/concrete/src/Page/Page.php

```diff
@@ -2836,6 +2836,7 @@ public function add($pt, $data, $template = false)
             $pkgID = 0;
         }
 
+        $data['name'] = null;
```

Comment:
`$data['name'] = $data['name'] ?? null;` would be better here since data is passed in.